schemas/webpagetest/webpagetest-run/webpagetest-run.1.parquetmr.txt

```diff
@@ -0,0 +1,19 @@
+message webpagetest-run {
+  required binary appName (UTF8);
+  required binary channel (UTF8);
+  required binary connection (UTF8);
+  required binary domain (UTF8);
+  required binary platform (UTF8);
+  required binary runner (UTF8);
+  required int32 runId;
```

Comment:
Could this be a string? I'm thinking we'd use `${BUILD_TAG}` from Jenkins, which equates to `jenkins-${JOB_NAME}-${BUILD_NUMBER}`, for example: "jenkins-wpt/google-123".

Reply:
Yes. We can also drop `runner` since example value you gave seems to be containing some Jenkins instance identifier `jenkins-wpt`.

Reply:
`runner` might still be useful as we _could_ have multple Jenkins instances. The "jenkins" in "jenkins-wpt/google-123" is hardcoded and does not distinuish the instance.